content/en/docs/concepts/services-networking/service.md

```diff
@@ -203,7 +203,9 @@ subsets:
 The name of the Endpoints object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
-{{< note >}} The name of the Endpoint must be the same as the name of the Service. {{< /note >}}
+When you create an [Endpoints](docs/reference/kubernetes-api/service-resources/endpoints-v1/)
+object for a Service, you set the name of the new Endpoints to be the same as the
```

Comment:
I would say "Endpoints object" in every mention of the object. Otherwise it reads a bit strangely. Nit, really.

Reply:
Done in https://github.com/kubernetes/website/pull/33197/commits/65dbc312c73cbe7873dd63aac60d9c2d0198052c